### ISSUE TITLE
An options button would be nice.

### ISSUE THREAD

**Issue by Heebe** [torch2424/Fight#11]
So that once you pause the game you can directly change things such as SFX on/off, Music on/off, button mapping, and graphics options so that you don't have to exit out for that.

**Comment by torch2424** [comment]
Yeeyeyyyeyeyeyeyeyeyeyeyeyeyeyeyeyeyeyeye!

**Comment by torch2424** [comment]
Thanks for doing this dude, honestly anything at all that even seems obvious that is missing, make an issue for it, because this not only shows me bugs, but a to-do list you know? @Heebe

**Comment by Heebe** [comment]
Yeah you're welcome, I got it. All in hopes to be a co-creator of a game with you next time rather than just help work.
